subtests/docker_cli/login/login.py

```diff
@@ -161,6 +161,9 @@ def docker_login(self, password=None):
         """
         if password is None:
             password = self.sub_stuff['passwd']
+        # FIXME: --email option is deprecated in 1.12, and will be
+        # removed in 1.13. We still need it for <= 1.10; otherwise
+        # docker prompts from stdin, and we hang.
```

Comment:
Ugg, that's annoying.  Does it give a warning in 1.12 but still accept the option?  if so, this can be fixed by a > 1.12 version-check.  No prob. leaving it as a FIXME here for now.